docs/modules/ROOT/pages/install-clc.adoc

```diff
@@ -1,4 +1,128 @@
 = Installing Hazelcast CLC
-:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS and Linux environments.
+:description: The Hazelcast Command-Line Client (CLC) is available to install in macOS, Linux and Windows environments.
+
+// See https://docs.hazelcast.com/hazelcast/5.2-snapshot/clients/clc#installing-the-hazelcast-clc
+
+{description} You can install CLC using either of the following ways:
+
+* Homebrew (recommended)
+* Installer script
```

Comment:
It seems that we have specific instructions and options, depending on the OS. I think we should focus on the operating system. Users will want to see all the options for their OS before choosing a method.

Maybe the headings should be Installing on Windows, Installing on Linux, Installing on macOS. That way, users can see in the ToC that we support their OS and they can jump to the content they need. Then we don't need the 'Before you begin'. For example, under 'Installing on Linux' we might have tabs for Homebrew, Install Script, and Manual. The first step for Homebrew could be:

' 1. Make sure that you have Homebrew installed.' with a link to the Homebrew install page and a code snippet that shows how to check if it's installed.